src/styles/_catalog.scss

```
@@ -1,22 +1,33 @@
+$card-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width) ;
+  justify-content: center;
+  place-items: center;
+  gap: 48px;
+
+  padding-bottom: 50px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
 
   &__card {
     display: flex;
     justify-content: center;
```

replace flex to grid
```suggestion
    display: grid;
    justify-content: center;
```